src/crossover.rs

```diff
@@ -68,25 +66,37 @@ impl Serializable<{ 64 + PoseidonCipher::SIZE }> for Crossover {
 }
 
 impl Crossover {
-    /// Returns a hash represented by `H(value_commitment)`
-    pub fn hash(&self) -> BlsScalar {
-        let value_commitment = self.value_commitment().to_hash_inputs();
+    /// Represent the crossover as a sequence of scalars to be used as input for
+    /// sponge hash functions
+    pub fn to_hash_inputs(
+        &self,
+    ) -> [BlsScalar; 3 + PoseidonCipher::cipher_size()] {
+        let mut inputs = [BlsScalar::zero(); 3 + PoseidonCipher::cipher_size()];
```

Comment:
Where this `3` comes from? You should either add a clear comment or one or more const with a clear name.

Reply:
We have inside this function 3 scalars + whatever comes from the poseidon cipher:

https://github.com/dusk-network/phoenix-core/blob/ba2450df3d0045d3dd7a0ecfab0b5a173cfe1f49/src/crossover.rs#L71-L81

Reply:
As discussed during the stand down, just a comment in this method would suffice about where `3` is coming from (same for `Message`)

Reply:
Fixed